bgpd/bgp_attr.c

```diff
@@ -2211,6 +2213,17 @@ static bgp_attr_parse_ret_t bgp_attr_psid_sub(int32_t type,
 
 	/* Placeholder code for the Originator SRGB type */
 	else if (type == BGP_PREFIX_SID_ORIGINATOR_SRGB) {
+		if (STREAM_READABLE(peer->curr) < length
+		    || length != BGP_PREFIX_SID_IPV6_LENGTH) {
+			flog_err(EC_BGP_ATTR_LEN,
+				 "Prefix SID Originator SRGB length is %" PRIu16
+				 " instead of %u",
+				 length, BGP_PREFIX_SID_IPV6_LENGTH);
```

Comment:
here BGP_PREFIX_SID_ORIGINATOR_SRGB_LENGTH as well?

Reply:
dont drink and code